src/gcodeExport.cpp

```diff
@@ -602,10 +630,10 @@ void GCodeExport::writeTravel(int x, int y, int z, double speed)
     CommandSocket::sendLineTo(travel_move_type, Point(x, y), display_width);
 
     *output_stream << "G0";
-    writeFXYZE(speed, x, y, z, current_e_value);
+    writeFXYZE(speed, x, y, z, current_e_value, PrintFeatureType::MoveCombing);
```

Comment:
I can probably change it if you want, just takes some figuring out where the travels to new print features are made :-)

I meant it like a global thing tho. You can see that travel time goes down 50% when you change infill pattern to zigzag. You see how these settings influence each other.